File: web/app/MealFood.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class MealFood extends Model
{

  protected $fillable=[
    'passenger_id', 'food_id', 'meal_id'
  ];

  public function passenger(){
    return $this->belongsTo('App\Passenger', 'id', 'passenger_id');
  }

  public function food(){
    return $this->hasOne('App\Food', 'id', 'food_id');
  }
}

File: android/app/src/main/java/tut/flightbookingsystem/model/MealFood.java
package tut.flightbookingsystem.model;

import java.io.Serializable;

public class MealFood implements Serializable {
    public long id;
    public long passenger_id;
    public long food_id;
    public Food food;
    public String created_at;
    public String updated_at;
}
